src/main/scala/it/pagopa/interop/authorizationmanagement/server/impl/Main.scala

```diff
@@ -23,7 +23,7 @@ import scala.concurrent.ExecutionContextExecutor
 
 object Main extends App with Dependencies {
 
-  val logger: Logger = Logger(this.getClass())
+  val logger: Logger = Logger(this.getClass)
```

Comment:
I usually get a deprecation warning when I remove parenthesis since scala 3 won't support it

Reply:
Didn't know that.
[Here](https://docs.scala-lang.org/scala3/book/methods-most.html) seems that the feature is still supported. Is it outdated?

Reply:
I'll research a bit. Ignore this comment for the moment.
(btw my style is to call the java stuff with parenthesis in case. You know... sometimes <strike>shit</strike> java happens)